backend/api/battles/serializers.py

```diff
@@ -94,16 +92,16 @@ def validate(self, attrs):
     def create(self, validated_data):
         return Battle.objects.create(**validated_data)
 
-    def update(self, instance, validated_data):
+    def update(self, instance, validated_data):  # noqa
```

Comment:
@filipeximenes, I added `# noqa` here because I was receiving  `pylint - inconsistent-return-statements` because of the first return. If you know a way of fix this without `# noqa`, please tell me.

Reply:
It will probably work if you do `return None` instead of `return`